package.json

```
@@ -11,8 +11,7 @@
     "start": "gatsby develop",
     "start:refresh": "ENABLE_GATSBY_REFRESH_ENDPOINT=true yarn start",
     "refresh": "curl -X POST http://localhost:8000/__refresh",
-    "import:contentful:demo": "contentful space import --config .contentful/import-demo-config.json",
-    "import:contentful:model": "contentful space import --config .contentful/import-model-only-config.json",
+    "import:contentful:demo": "node .contentful setup.js",
```

Should we just call this `setup` now since the script does a few other things?

```suggestion
    "setup": "node .contentful setup.js",
```